indica/datatypes.py

```diff
@@ -196,4 +198,6 @@
     "absorbed_ohmic_power": ("$P_{ohm, abs}$", "power"),
     "safety_factor": ("q", "none"),
     "parallel_conductivity": (r"$\sigma$", "conductivity"),
+    "frequency": ("$omega$", "frequency"),
```

Comment:
What sort of frequency? Worth being more specific since the LongName comes out in the plot as well and explains what this variable really is